Werewolf stream8 erkin bjs2 - 67102

Erik18999 opened the pull request:
BJS2-67110 | Создать структуру БД
BJS2-67109 | Обработчик исключений
BJS2-67108 | Работа с базой(HashRepository)
BJS2-67103 | Генерация списка хэшей
BJS2-67105 | Генерация хэша
BJS2-67104 | Джоб для освобождения неиспользуемых хэшей BJS2-67111 | Создание оригинальной ссылки
BJS2-67106 | Кэширование свободных хэшей
BJS2-67107 | Создание короткой ссылки

Commit:
BJS2-67110 | Создать структуру БД
BJS2-67109 | Обработчик исключений
BJS2-67108 | Работа с базой(HashRepository)
BJS2-67103 | Генерация списка хэшей
BJS2-67105 | Генерация хэша
BJS2-67104 | Джоб для освобождения неиспользуемых хэшей
BJS2-67111 | Создание оригинальной ссылки
BJS2-67106 | Кэширование свободных хэшей
BJS2-67107 | Создание короткой ссылки

zhagot commented on src/main/java/faang/school/urlshortenerservice/cache/HashCache.java:
```diff
@@ -0,0 +1,6 @@
+
+public interface HashCache {
+
+    public String getHash();
```
А зачем тебе в интерфейсе public? ;-)

Commit:
BJS2-67110 | Создать структуру БД
BJS2-67109 | Обработчик исключений
BJS2-67108 | Работа с базой(HashRepository)
BJS2-67103 | Генерация списка хэшей
BJS2-67105 | Генерация хэша
BJS2-67104 | Джоб для освобождения неиспользуемых хэшей
BJS2-67111 | Создание оригинальной ссылки
BJS2-67106 | Кэширование свободных хэшей
BJS2-67107 | Создание короткой ссылки
Refactoring
